ts/src/huobi.ts

```diff
@@ -5272,7 +5272,7 @@ export default class huobi extends Exchange {
             for (let j = 0; j < currencies.length; j++) {
                 const currency = currencies[j];
                 const currencyId = this.safeString (currency, 'currency');
-                const code = this.safeCurrencyCode (currencyId, 'currency');
+                const code = this.safeCurrencyCode (currencyId);
```

Comment:
@blair You're right, I have no idea why we had `currency` as the second argument...